tests/contrib/actions/slack/test_slack_message_manager.py

```diff
@@ -39,3 +51,12 @@ def test_api_call_with_error_should_log_error_msg(mocker, logger_error):
 
     assert logger_error.call_count == 1
     assert error_msg in logger_error.call_args_list[0][0]
+
+
+def test_load_recipients_list_from_crawler_settings():
+    settings = {
+        "SPIDERMON_SLACK_RECIPIENTS": "foo,bar",
```

Comment:
I believe it is worthy to include a test case for the already existing behaviour (`["foo", "bar"]`) , so we can be sure we didn't break anything. You can use `@pytest.mark.parametrize` to avoid duplicating the test code.